core/modules/simpletest/tests/image.test

```diff
@@ -323,14 +322,14 @@ class ImageToolkitGdTestCase extends BackdropWebTestCase {
       $operations += array(
         'rotate_5' => array(
           'function' => 'rotate',
-          'arguments' => array(5, 0xFF00FF), // Fuchsia background.
+          'arguments' => array(5, '0xFF00FF'), // Fuchsia background.
```

Comment:
Interesting, this change does seem to be necessary. Without the quotes the tests fail.